lib/platform/azure/index.ts

```diff
@@ -840,3 +841,10 @@ export async function deleteLabel(
 export function getVulnerabilityAlerts(): Promise<VulnerabilityAlert[]> {
   return Promise.resolve([]);
 }
+
+// istanbul ignore next
+export function commitFiles(
```

Comment:
Do we really need those simple wrapper?
Can't we do something like `platform.commitFiles ? platform.commitFiles(config) : git.commitFiles(config)` where required?